src/Module1/Task1-2.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Write a program which should do the following:
+ *
+ * [*] Read the content of csv file from ./csv directory.
+ * [*] Use the csvtojson package (https://github.com/Keyang/node-csvtojson) to convert csv file to json object.
+ * [*] Write the csv file content to a new txt file. Use the provided format.
+ * [*] Do not load all the content of the csv file into RAM via stream (read/write file content line by line).
+ * [*] In case of read/write errors, log them in the console.
+ * [*] The program should be started via npm script using nodemon (i.e. npm run task2)*
+ *
+ * */
+
+import fs from 'fs';
+import csvtojson from 'csvtojson';
+
+const csvFilePath = './csv/nodejs-hw1-ex1.csv';
```

Comment:
I would suggest using smth like path.resolve

Reply:
Thank you! I'll keep in mind for further implementations